docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,109 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. This may be appropriate in some instances, but removing those resources will be
+necessary in others. This can be achieved by utilizing one of the two following methods.
+
+The first would be to utilize a clean deployment with a new cluster profile. The previous version could be deleted once
+the new cluster and cluster profile are deployed. The challenge, however, is when particular resources are in use.
+
+The second option is to use `kubectl` to scale the resource count to 0. While deleting the resource using something like
+K9s is possible, Palette will recreate it immediately.
+
+### Limitations
+
+:::warning
+
+Once you upgrade your cluster to a new Kubernetes version, you will not be able to downgrade. We recommend that, before
+upgrading, you review the information provided in the
+[Kubernetes Upgrades](../../integrations/kubernetes-support.md#kubernetes-upgrades) section.
+
+:::
+
+- You cannot update a cluster while its status is still **Provisioning**.
+
+- You cannot change the Container Network Interface (CNI) of an Edge cluster. You must delete and re-create the cluster
+  instead. For more information about creating an Edge cluster, refer to
+  [Create Cluster Definition](../edge/site-deployment/cluster-deployment.md).
+
+- Avoid skipping minor versions when upgrading the Kubernetes version of a cluster. Refer to the documentation of your
+  Kubernetes distribution for upgrade guidance and follow the recommended upgrade paths.
+
+  - For PXK and PXK-E, refer to
+    [Upgrade Kubeadm Clusters](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
+  - For K3s, refer to [K3s Upgrades](https://docs.k3s.io/upgrades#version-specific-caveats)
+  - For RKE2, refer to [RKE2 Manual Upgrades](https://docs.rke2.io/upgrades/manual_upgrade)
```

Comment:
@Linus-SpectroCloud You can use a "global" Limitations section for an article, so I would put a Limitations (header 2) at the beginning of the article that covers both How-tos.

The rest of the article headers are all in alignment with our guidance though.